app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -134,13 +130,58 @@ fun MessageCard(
 
                 PrimaryButton(
                     text = buttonText,
+                    textColor = colors.buttonTextColor,
+                    backgroundColor = colors.buttonColor,
                     onClick = onClick,
                 )
             }
         }
     }
 }
 
+/**
+ * Wrapper for the color parameters of [MessageCard].
+ *
+ * @param backgroundColor The background [Color] of the message.
+ * @param titleTextColor [Color] to apply to the message's title, or the body text when there is no title.
+ * @param messageTextColor [Color] to apply to the message's body text.
+ * @param iconColor [Color] to apply to the message's icon.
+ * @param buttonColor The background [Color] of the message's button.
+ * @param buttonTextColor [Color] to apply to the button text.
+ */
+data class MessageCardColors(
+    val backgroundColor: Color,
+    val titleTextColor: Color,
+    val messageTextColor: Color,
+    val iconColor: Color,
+    val buttonColor: Color,
+    val buttonTextColor: Color,
+) {
+    companion object {
+
+        /**
+         * Builder function used to construct an instance of [MessageCardColors].
+         */
+        @Composable
+        fun buildMessageCardColors(
+            backgroundColor: Color? = null,
+            titleTextColor: Color? = null,
+            messageTextColor: Color? = null,
+            iconColor: Color? = null,
+            buttonColor: Color? = null,
+            buttonTextColor: Color? = null,
+        ): MessageCardColors =
+            MessageCardColors(
+                backgroundColor = backgroundColor ?: FirefoxTheme.colors.layer2,
+                titleTextColor = titleTextColor ?: FirefoxTheme.colors.textPrimary,
+                messageTextColor = messageTextColor ?: FirefoxTheme.colors.textSecondary,
+                iconColor = iconColor ?: FirefoxTheme.colors.iconPrimary,
+                buttonColor = buttonColor ?: FirefoxTheme.colors.actionPrimary,
+                buttonTextColor = buttonTextColor ?: FirefoxTheme.colors.textActionPrimary,
```

Comment:
This seems like a pretty common comment at this point 🤭, but let's add those fallback color values as the default values so we don't have to make the parameters nullable.

Reply:
Sure thing. I think I was over-coding this, when it was only the Pocket backplating that required some of the nullable logic.